en/setup/cssCustomisation.md

```diff
@@ -0,0 +1,16 @@
+# Customise the look of JabRef using CSS
+The look of JabRef can be customised by importing custom CSS files. Imported themes, as well as the two basic themes _Light_ and _Dark_, can be exported from JabRef in the form of CSS files.
```

Comment:
Please separate heading and content by a single line. --> https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md022